apisix/ssl/router/radixtree_sni.lua

```diff
@@ -208,7 +208,9 @@ function _M.match_and_set(api_ctx)
     local sni
     sni, err = ngx_ssl.server_name()
     if type(sni) ~= "string" then
-        return false, "failed to fetch SSL certificate: " .. (err or "not found")
+        local advise = "please check if the client requests via IP or uses an outdated protocol" ..
+            ". If you need to report an issue, provide a packet capture file of the TLS handshake."
```

Comment:
I think it would be better to align the tips of `advise`, like this:

```
local advise = "please check if the client requests via IP or uses an outdated protocol." ..
               "If you need to report an issue, provide a packet capture file of the TLS handshake."
```

Reply:
Changed.